public/scripts/personas.js

```diff
@@ -572,6 +582,23 @@ export function setPersonaDescription() {
         .find(`option[value="${power_user.persona_description_role}"]`)
         .prop('selected', String(true));
     $('#persona_lore_button').toggleClass('world_set', !!power_user.persona_description_lorebook);
+
+    // Load pronoun values from current persona
+    const personaData = power_user.persona_descriptions?.[user_avatar];
+    if (personaData) {
+        $('#persona_pronoun_subjective').val(personaData.pronounSubjective || '');
+        $('#persona_pronoun_objective').val(personaData.pronounObjective || '');
+        $('#persona_pronoun_pos_det').val(personaData.pronounPosDet || '');
+        $('#persona_pronoun_pos_pro').val(personaData.pronounPosPro || '');
+        $('#persona_pronoun_reflexive').val(personaData.pronounReflexive || '');
+    } else {
+        $('#persona_pronoun_subjective').val('');
+        $('#persona_pronoun_objective').val('');
+        $('#persona_pronoun_pos_det').val('');
+        $('#persona_pronoun_pos_pro').val('');
+        $('#persona_pronoun_reflexive').val('');
+    }
```

Comment:
Same here, no need to do the if/else, you can just make the optional `personaData?.pronounSubjective` access.